ads/revcontent.md

```diff
@@ -0,0 +1,41 @@
+<!---
+Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Revcontent
+
+## Example
+
+```html
+<amp-ad width=300 height=1000
+        type="revcontent"
+        data-wrapper="rcjsload_2ff711"
+        data-endpoint="trends.revcontent.com"
+        data-ssl="true"
+        data-id="203">
+
+</amp-ad>
+```
+
+## Configuration
+
+For semantics of configuration, please see ad network documentation.
+
+Supported parameters:
+
+- wrapper. Must be a valid dom.div id string
```

Comment:
Our *readme.md* file was based off of another template from several revisions ago, it will update to match layout of the other readme files.